package.json

```diff
@@ -20,6 +21,18 @@
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "^5.2.1",
     "@reduxjs/toolkit": "^1.2.3",
+    "@types/prosemirror-commands": "^1.0.4",
```

Comment:
Can these be moved to `devDepencies`?

Reply:
Done. I removed all prosemirror type dependencies and added @types/prosemirror-state to devDependencies.